github/ci/prow/files/jobs/kubevirt/kubevirt-csi/kubevirt-csi-presets.yaml

```diff
@@ -0,0 +1,32 @@
+presets:
+- labels:
+    preset-kubevirt-csi-quay-credential: "true"
+  env:
+  - name: QUAY_USER
+    value: /etc/kubevirtci-cred/username
+  - name: QUAY_PASSWORD
+    value: /etc/kubevirtci-cred/password
+  volumes:
+  - name: kubevirtci-cred
+    secret:
+      defaultMode: 0400
+      secretName: kubevirtci-quay-credential
+  volumeMounts:
+  - name: kubevirtci-cred
+    mountPath: /etc/kubevirtci-cred
+    readOnly: true
+
+- labels:
+    preset-kubevirt-csi-installer-pull-token: "true"
```

Comment:
I don't see this preset used in the job?

Reply:
removed the file as you suggested